### DISCUSSION TITLE
How to re-use sql-talk-app with an existant bigquery database ?

### DISCUSSION THREAD

**Discussion by ricobomb** [community/community#122677] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi and congratulations for your great sql-talk-app ! it's a perfect sample for the long waiting use case I'm searching for my company. I'm not a professional developer but i would like to reuse the code and just change the setup.sh and the app.py for using my already existant bigquery database. Can you please tell me what instructions to change in these 2 files ?

**Comment by ebndev** [comment]
Hi @ricobomb,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from sql-talk-app. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Comment by ebndev** [comment]
Hi again @ricobomb!

If you don't mind, would you mind letting us know how you initially ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your discussion and providing the link to the project's repository? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board, if that’s where you intended to post.

Any insight would be helpful, thanks!

**Reply by ebndev** [reply]
Never mind, found the issue! Thanks again for your time and being a part of the GitHub Community 😄
